test/import/FSHImporter.Alias.test.ts

```diff
@@ -355,7 +371,55 @@ describe('FSHImporter', () => {
       expect(rule.value).toEqual(
         new FshCode('foo', 'BAD|LOINC').withFile('Loinc.fsh').withLocation([6, 10, 6, 22])
       );
+      expect(loggerSpy.getAllMessages('error')).toHaveLength(1);
       expect(loggerSpy.getLastMessage('error')).toMatch(/BAD\|LOINC cannot include "\|"/);
     });
+
+    it('should log an error when an alias contains unsupported characters', () => {
+      // The only supported characters in Aliases are ASCII letters, numbers, _, -, .
+      const input = leftAlign(`
+      Alias: B@dAlias = http://example.org
+
+      Profile: ObservationProfile
+      Parent: Observation
+      * code = B@dAlias#foo
+      `);
+
+      const results = importText([new RawFSH(input, 'Loinc.fsh')]);
+      expect(results.length).toBe(1);
+      const rule = results[0].profiles.get('ObservationProfile').rules[0] as AssignmentRule;
+      // The B@dAlias alias does not resolve, since it is not created
+      expect(rule.value).toEqual(
+        new FshCode('foo', 'B@dAlias').withFile('Loinc.fsh').withLocation([6, 10, 6, 21])
+      );
+      expect(loggerSpy.getAllMessages('error')).toHaveLength(1);
+      expect(loggerSpy.getLastMessage('error')).toMatch(/B@dAlias includes unsupported characters/);
+    });
+
+    it('should log an error when an alias contains unsupported characters and starts with $', () => {
+      // The only supported characters in Aliases are ASCII letters, numbers, _, -, .
```

Comment:
These two extra tests are nice. Could we get a test for an alias that has all the special allowed characters too? Like `Foo-foo_McFooferton_Jr._the_3rd`?

Reply:
Good idea! Added in [c97868b](https://github.com/FHIR/sushi/pull/1429/commits/c97868b063808f28e7764bd3c0126f8ac11d209b).